src/components/CategoryVideos/CategoryVideos.styles.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+
+import { colors, media, sizes } from '@/shared/theme'
+
+export const Container = styled.div`
+  margin-top: ${sizes(16)};
+`
+
+export const ControlsContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr auto;
+  gap: ${sizes(4)};
+  align-items: center;
+  padding-bottom: ${sizes(4)};
+  border-bottom: 1px solid ${colors.gray[700]};
+
+  ${media.md} {
+    grid-template-columns: auto 168px 1fr 241px;
```

Comment:
Where are those values coming from?

Reply:
This is the size of the dropdown sections on MD breakpoint:
https://www.figma.com/file/UdBMxrLspjcf6IVYYJS8YP/[Page]-Discover-and-Video-category?node-id=252%3A93102

Reply:
Could you help me understand what you mean by dropdown sections? Are you sure that Figma specifies this values as fixed as you currently do? I'm pretty sure the values depend on something else than a hardcoded value

Reply:
Maybe @toiletgranny might be able to clarify here, but the language dropdown and the sort by dropdown don't seem to be elements that scale or shrink in this MD breakpoint, in any case it was the searchbar the element that used to take free available space but is not there anymore

Reply:
I still don't understand what this value are supposed to style. As far as I can tell, selects in the design just follow automatic width as all the other selects? I don't think they need to be explicitly sized like this?

Reply:
@eldiegod Still waiting on response for this

Reply:
They are the reserved space for the language dropdown and the sort by label + dropdown, I'm not sure what are you suggesting here instead? making them grow or shrink doesn't seem to fit design